bin/interpreter.sh

```diff
@@ -86,6 +89,9 @@ ZEPPELIN_SERVER=org.apache.zeppelin.interpreter.remote.RemoteInterpreterServer
 INTERPRETER_ID=$(basename "${INTERPRETER_DIR}")
 ZEPPELIN_PID="${ZEPPELIN_PID_DIR}/zeppelin-interpreter-${INTERPRETER_ID}-${ZEPPELIN_IDENT_STRING}-${HOSTNAME}.pid"
 ZEPPELIN_LOGFILE="${ZEPPELIN_LOG_DIR}/zeppelin-interpreter-"
+if [[ ! -z "$INTERPRETER_NAME" ]]; then
+    ZEPPELIN_LOGFILE+="${INTERPRETER_NAME}-"
+fi
```

Comment:
I am thinking that we could pass interpreter name as environment variable `INTERPRETER_DIR` to the interpreter process, so that we don't need to change `interpreter.sh`